data-raw/move_inaug_results.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
This file appears to be identical to `move_inaug_results.sh`. Is that intentional?